package.json

```diff
@@ -9,7 +9,7 @@
     "workos"
   ],
   "volta": {
-    "node": "19.9.0",
+    "node": "20.15.0",
```

Comment:
Any specific reason for the bump here?

Reply:
Unintentional, but I couldn't run tests with the current versions due to a conflict. I can undo this.